### ISSUE TITLE
Pass paper-date-picker options through

### ISSUE THREAD

**Issue by mh166** [Collaborne/paper-date-picker-item#6]
It should be possible to pass options directly to the `<paper-date-picker>` element within the dialog. The only colliding property is `date`, which is no problem as it is bound to the date picker anyways, for obvious reasons. :wink:

So for example the following should change the `<paper-date-picker>`'s locale:

``` html
<paper-date-picker-item placeholder="Datum auswählen" locale="de"></paper-date-picker-item>
```

This would be of great help, not only in order for l10n to be applied, but also to adjust any of the numerous properties available to the date picker.

**Comment by ronnyroeller** [comment]
Good idea, @mh166. Feel free to provide a PR for the properties that you would like to be passed through.

**Comment by ankon** [comment]
At least `locale` is now available: https://github.com/Collaborne/paper-date-picker-item/issues/9
